docker-compose.yml

```diff
@@ -7,10 +7,11 @@ services:
     - DEBUG=cybernaut:*
     volumes:
     - ./examples/dist:/opt/cybernaut-tests
+    - ./tests:/opt/cybernaut-tests
```

Comment:
@clebert If you use `docker-compose` approach and want to run your own tests inside the docker container, there needs to be a way to mount your tests. That's why I thought it would be nice to have such a folder already mounted.